### PR TITLE
Added mascots

### PR DESCRIPTION
**Pull request by kevinlu1248**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -266,46 +266,64 @@ <h3>
...
-                                    <a href="#">
-                                        <img src="img/case_study/2.png" alt="" />
-                                    </a>
-                                </div>
-                                <div class="subheading">
-                                    <h4><a href="#">Custom Website</a></h4>
-                                    <p>UI/UX, Design</p>
-                                </div>
+                            <div class="subheading">
+                                <!--                                    https://intellij-support.jetbrains.com/hc/en-us/community/posts/207242529-Debugging-with-Chrome-no-longer-works
```
Comment (kx-chen) [inline index.html]: What's with this comment about Intellij debugging?

File: index.html [hunk 2]
```diff
@@ -346,14 +360,52 @@ <h3 class="page-heading">What is Hackcouver?</h3>
...
+        <div class="meet-our-team mx-3 px-3">
+            <div class="container text-center no-border">
+                <div class="row">
+                    <div class="col-4">
+                        <div class="card">
+                            <img src="..." class="card-img-top" alt="...">
+                            <div class="card-body">
+                                <h5 class="card-title">Sam Mitchell</h5>
+                                <h5 class="card-subtitle">Human Resources</h5>
+                                <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
```
Comment (kx-chen) [inline index.html]: Still WIP? Are we adding the rest of the team as well?

File: js/owl.carousel.min.js [hunk 3]
```diff
@@ -3,5 +3,1751 @@
...
\ No newline at end of file
+!(function(a, b, c, d) {
+    function e(b, c) {
```
Comment (kx-chen) [inline js/owl.carousel.min.js:7]: How come this entire file got formatted?

**Review by kx-chen — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (kevinlu1248) [reply index.html]: I will delete that

File: index.html (hunk 2 above)
Comment (kevinlu1248) [reply index.html]: Yes this was just a template

File: js/owl.carousel.min.js (hunk 3 above)
Comment (kevinlu1248) [reply js/owl.carousel.min.js:7]: I will revert it

File: .idea/hackcouver.github.io.iml [hunk 4]
```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kx-chen) [inline .idea/hackcouver.github.io.iml:1]: `.idea/` needs to be removed again.

**Review by kx-chen — CHANGES_REQUESTED** [review]
Let's get this merged first. This branch needs to pull in the changes from `new-site`, it's behind by a few commits. Other than that and the comments, it looks good.

File: index.html (hunk 1 above)
Comment (kevinlu1248) [reply index.html]: fixed

**Pull request closed without merging**